pkg/virt-handler/device-manager/device_controller_test.go

```diff
@@ -26,7 +26,7 @@ type FakePlugin struct {
 	Error      error
 }
 
-func (fp *FakePlugin) Start(_ chan struct{}) (err error) {
+func (fp *FakePlugin) Run(_ <-chan struct{}) (err error) {
```

Comment:
Why `Run()`?

Since `fp.Starts` maybe `Start()` is more intuitive here, or maybe changing to `fp.Running`